Description: Fix elisp install location
Author: David Caldwell <david@porkrind.org>

Index: commit-patch/Makefile
===================================================================
--- commit-patch.orig/Makefile	2013-05-15 20:43:23.000000000 -0700
+++ commit-patch/Makefile	2013-05-15 20:43:23.000000000 -0700
@@ -36,9 +36,9 @@
 install: $(ALL)
 	mkdir -p "$(PREFIX)/bin"
 	mkdir -p "$(PREFIX)/share/man/man1"
-	mkdir -p "$(PREFIX)/share/emacs/site-lisp"
+	mkdir -p "$(PREFIX)/share/emacs/site-lisp/commit-patch"
 	mkdir -p "$(PREFIX)/share/doc/commit-patch"
 	cp -a $(BIN)   "$(PREFIX)/bin"
 	cp -a $(MAN)   "$(PREFIX)/share/man/man1"
-	cp -a $(ELISP) "$(PREFIX)/share/emacs/site-lisp"
+	cp -a $(ELISP) "$(PREFIX)/share/emacs/site-lisp/commit-patch"
 	cp -a $(DOC)   "$(PREFIX)/share/doc/commit-patch"
