Description: Fix elisp install location
Author: David Caldwell <david@porkrind.org>

--- commit-patch-2.3.orig/Makefile
+++ commit-patch-2.3/Makefile
@@ -32,9 +32,9 @@ PREFIX=/usr/local
 install: $(ALL)
 	mkdir -p "$(PREFIX)/bin"
 	mkdir -p "$(PREFIX)/share/man/man1"
-	mkdir -p "$(PREFIX)/share/emacs/site-lisp"
+	mkdir -p "$(PREFIX)/share/emacs/site-lisp/commit-patch"
 	mkdir -p "$(PREFIX)/share/doc/commit-patch"
 	cp -a $(BIN)   "$(PREFIX)/bin"
 	cp -a $(MAN)   "$(PREFIX)/share/man/man1"
-	cp -a $(ELISP) "$(PREFIX)/share/emacs/site-lisp"
+	cp -a $(ELISP) "$(PREFIX)/share/emacs/site-lisp/commit-patch"
 	cp -a $(DOC)   "$(PREFIX)/share/doc/commit-patch"
